### ISSUE TITLE
Two hours missing from stats on 13th March 2016, daylight savings day

### ISSUE THREAD

**Issue by jlazic** [piwik/piwik#9949]
Hi,

I have just noticed that 2h and 23h stats are missing on "Visits by Server Time" graph in Piwik 2.16.0 for date 2016-03-13. This was daylight savings day for USA. If one hour of stats was missing I would attribute that to DST, but why would two hours of stats be gone?
Both server and PHP (5.6.17) have timezone set to UTC, while tracked sites have America/Detroit set as timezone. I have already invalidated reports for 2016-03-13 and recreated them, but that did not help.
All of web servers and database server have system same timezone settings (UTC), while MySQL has global.time_zone set to SYSTEM.

![Missing visits](http://i.imgur.com/8zT3UpR.png)

If you need more info let me know and I will try provide it.

**Comment by tsteur** [comment]
Can you see visits for 23h in visitor log? It looks like the 23h is related to another issue.

**Comment by jlazic** [comment]
Yes, there are logged visits for 23h. Also, I just noticed that in Visits per local time graph there is no data missing. Hope this helps.
![Visits per local time](http://i.imgur.com/wqZJSCp.png)

**Comment by tsteur** [comment]
For piwik.org tracking we have similar issue on `2016-03-27` and `23h`. We correctly get data for `2h` though. So can you maybe check whether there is any data for 2am in the visitor log (server time)?

**Comment by jlazic** [comment]
Field 'visit_first_action_time' in 'piwik_log_visit' table is what we call server time? If so then - yes, there are logged visits for 2h.

**Comment by tsteur** [comment]
I think in this case we need to look for `visit_last_action_time`

**Comment by jlazic** [comment]
Answer is still - yes, there are logged visits for both `visit_first_action_time` and `visit_last_action_time`.
